README.md

```diff
@@ -19,7 +19,7 @@ are unit tested and should be working as long as the build status is passing.
 git clone https://github.com/cta-observatory/cta-lstchain.git
 cd cta-lstchain
 conda env create -f environment.yml
-source activate cta
+source activate lst-dev
```

Comment:
while at it, I think the latest version of conda should be `conda activate lst-dev` , no ?

Reply:
thanks for spotting that !